pyproject.toml

```diff
@@ -32,31 +32,29 @@ dependencies = [
     "argcomplete >=3.1.4",
     "nest-asyncio >=1.5.6",
     "mango-agents-assume >=1.1.4-6",
-    "numpy < 2", # Pyomo Incompatible
+    "numpy >=1.26.4",
     "tqdm >=4.64.1",
     "python-dateutil >=2.8.2",
     "sqlalchemy >=2.0.9",
     "pandas >=2.0.0",
     "psycopg2-binary >=2.9.5",
     "pyyaml >=6.0",
     "pyyaml-include >=2.2a",
+    "pyomo >=6.8.0",
 ]
 
 [project.optional-dependencies]
 learning = [
     "torch >=2.0.1",
 ]
 optimization = [
```

Comment:
at this point we might as well rename the optimization to network since it only installs pypsa extra, what do you think?